src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -771,24 +777,48 @@ const WorkflowView = _.flow(
-                `Tables / ${rootEntityType}`
+            h(Collapse, { title: 'Output storage' }, [
+              h(Fragment, [
+                div({ style: styles.outputInfoLabel }, 'Output files will be saved to your workspace bucket:'),
+                div({ style: { display: 'flex', alignItems: 'center', margin: '0.5rem' } }, [
+                // TODO: why can't I use the 'RegionFlaggedContent' element here...?
+                  div({ style: { flex: 'none', display: 'flex', width: '1.5rem' } }, [
+                    h(TooltipTrigger, { content: div([`${locationType}: ${location} (${bucketRegionDescription})`]) }, [div(bucketRegionFlag)])
+                  ]),
+                  div({ style: { flex: 1 } }, [`gs://${bucketName}`])
+                ])
               ]),
-              `Fill in the attributes below to add or update columns in your data table`
+              h(Fragment, [
+                div({ style: { margin: '0.5rem 0', borderBottom: `1px solid ${colors.dark(0.55)}` } }),
+                div({ style: styles.outputInfoLabel }, 'Raw outputs will be viewable in the Data tab under:'),
+                div({ style: { display: 'flex', alignItems: 'center' } }, [
+                  div({ style: { flex: 'none', display: 'flex' } }, [
+                    h(Link, {
+                      href: Nav.getLink('workspace-data-bucket', { namespace, name: workspaceName, initialDataType: '__bucket_objects__' }),
+                      style: { marginRight: '0.5rem' }
+                    }, [icon('folder-open', { style: { marginRight: '0.5rem' } }), 'Files'])
+                  ]),
+                  div({ style: { flex: 1 } }, [
+                    ' / ',
+                    span({ style: styles.placeholder }, '<<unique ID for this submission>>'), ' / ',
+                    wdl ? wdl.match(/^\s*workflow ([^\s{]+)\s*{/m)[1] : span({ style: styles.placeholder }, 'workflow name'), ' / ',
+                    span({ style: styles.placeholder }, '<<unique ID for each workflow>>'), ' / ',
+                    span({ style: styles.placeholder }, '<<call name>>')
+                  ])
+                ])
+              ]),
+              !!rootEntityType && h(Fragment, [
+                div({ style: { margin: '0.5rem 0', borderBottom: `1px solid ${colors.dark(0.55)}` } }),
+                div({ style: styles.outputInfoLabel }, 'References to outputs can be written to'),
+                div({ style: { display: 'flex', alignItems: 'center' } }, [
+                  h(Link, {
+                    href: Nav.getLink('workspace-data-bucket', { namespace, name: workspaceName, initialDataType: 'localVariables' }),
+                    style: { marginRight: '0.2rem' }
+                  }, [icon('listAlt', { style: { marginRight: '0.5rem' } }), 'Tables']),
+                  ` / ${rootEntityType}`
+                ]),
+                'Fill in the attributes below to add or update columns in your data table'
+              ])
```

Comment:
TOL: This seems like a nice change overall, but given the complexity with regard to the Data page, perhaps this could have been split out as a separate PR.